pages/22.What's New/docs.en.md

```diff
@@ -13,6 +13,32 @@ taxonomy:
 ----------------------------
 In this section we'll highlight the new features for each major release of Mautic.  They are in version order beginning with the latest.
 
+##Version 3.0
+Mautic 3.0 introduces significant features and a number of enhancements. They are outlined as follows:
+
+###Hosting Requirements
+- Mautic 3.0 added support for PHP versions 7.2 and 7.3. The minimum PHP version is raised from version 5.6.19 to version 7.2.0.
```

Comment:
Support for 7.2 and 7.3 were available in 2.16 releases so I don't think we need to include that here. 

```suggestion
- Mautic 3.0 raised the minimum PHP version required from 5.6.19 to 7.2.0.
```